pages/views/examples/footnotes.njk

```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: Footnotes
+seoTitle: Auto-collected footnotes custom elements example based on ESL web components
```

Comment:
Based on other seoTitles this particular title should be rewritten like: "Custom auto-collected footnotes element examples based on ESL web components"